sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -199,25 +214,25 @@ async def create_item(
     @distributed_trace_async
     async def read_item(
             self,
-            item,  # type: Union[str, Dict[str, Any]]
-            partition_key,  # type: Any
-            **kwargs  # type: Any
+            item: Union[str, Dict[str, Any]],
+            partition_key: Any,
```

Comment:
As Travis mentioned - we should revisit the classification of `partition_key` as Any throughout.